ethcore/src/blockchain/config.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Blockchain configuration.
+use bloomchain::Config as BloomConfig;
+
+/// Blockchain configuration.
+#[derive(Debug)]
+pub struct Config {
```

Comment:
This option (`log_blooms`) can be configurable now. Syncing from scratch could be done with different number of `levels` and `elements_per_index`. There is no cli for it yet, but I'm willing to add it someday.